Library/Homebrew/missing_formula.rb

```diff
@@ -8,8 +8,8 @@
 module Homebrew
   module MissingFormula
     class << self
-      def reason(name, silent: false, show_info: false)
-        cask_reason(name, silent: silent, show_info: show_info) || blacklisted_reason(name) ||
```

Comment:
I changed the order of these because there was a test that was calling `cask_reason` before `blacklisted_reason`,  and as `Homebrew.args.argv` was nil because it's a test an Error was being thrown. Not sure if there's a more elegant way to deal with that but I'm open to suggestions.
 https://github.com/Homebrew/brew/blob/e0c645bab1a71ff97eaedecd5558946ce5c2e88d/Library/Homebrew/test/missing_formula_spec.rb#L6